examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/MainApplication.kt

```diff
@@ -29,7 +29,6 @@ class MainApplication : Application(), UpdatedCustomerInfoListener {
                 VmPolicy.Builder()
                     .detectLeakedClosableObjects()
                     .penaltyLog()
-                    .penaltyDeath()
```

Comment:
Intentional deletion. This was causing random crashes that made testing purchase tester almost impossible.

Reply:
Thanks for pointing that out the other day 💪

Reply:
@vegaro any context on why that was there?

Reply:
I've had this happen a few times... Usually uninstalling/reinstalling fixes the crashes for me. It would be great to understand the issue but yeah, I'm not sure it's worth to keep it